driftbase/api/messages.py

```diff
@@ -210,7 +210,16 @@ def get(self, exchange, exchange_id, queue, message_id):
 
 @endpoints.register
 def endpoint_info(*args):
-    return {
+    # TODO: This way of building templated URLs is very repetitive. I'm positive we can wrap this.
```

Comment:
Who will use this template? The system is actually quite restrictive on who can send things to each queue and exchange, so it might not be all that useful outside the pre-setup my_messages URL

Reply:
I may be missing something, but I'm not getting how this is relevant except for convenience in tests-writing.  Clients will only ever get from the 'players' exchange and only by parsing the messages from that can they get a specific message from a particular queue.

Reply:
I will remove the template. You're correct that it's only really relevant for testing.

Reply:
Done. I moved the `url_for` stuff into the test harness. Are we OK with using context handlers in this way, or should I be building the URL somehow else?